### PR TITLE
Features/add UI ux semantic

### PR DESCRIPTION
**Pull request by nomadjkeeee**
Add Semantic UI and basic style , fonts. Use diff cmd for install . Described this issues in trello card.

### CODE REVIEW

File: frontend/src/styles.sass [hunk 1]
```diff
@@ -1 +1,71 @@
...
+.input-dark
+    background: #282F37
+    border-radius: 50px
+
+
+//cards
+
+card
+    background-color: #282F37
+    border-radius: 4px
```
Comment (proskurnicha) [inline frontend/src/styles.sass:84]: Add basic colors separately like this:
image

**Review by proskurnicha — CHANGES_REQUESTED** [review]

File: frontend/src/app/app.component.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
 <!-- * * * * * * * to get started with your project! * * * * * * * * -->
 <!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
 
-<style>
+<!-- <style>
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/app.component.html]: Delete comments

File: frontend/src/app/app.component.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
+* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/app.component.html]: Looks like not a comment anymore

Comment (nomadjkeeee) [reply frontend/src/app/app.component.html]: Sorry. My false. I have already fixed this issue.

File: frontend/src/styles.sass (hunk 1 above)
Comment (nomadjkeeee) [reply frontend/src/styles.sass:84]: I had tried, but after when I built project ,  I saw this error 
*SassError: Undefined variable.
   
13 │   border: 1px solid $border-dark - last word mark underline.
But if i dont use defined color,  only define, project build and start.
I continue finding solution .

Comment (nomadjkeeee) [reply frontend/src/styles.sass:84]: Should I only define colors ? Or define and use in that style.sass ?

Comment (nomadjkeeee) [reply frontend/src/styles.sass:84]: Solved

**Comment by nomadjkeeee** [comment]
Add basic colors and fix commited area.

**Review by proskurnicha — APPROVED** [review]

**Pull request merged**